src/components/pipelines/PipelinesPage.tsx

```diff
@@ -24,10 +24,10 @@ const pipelineMarkdowns = {
             />
         </div>
     ),
-    rna_level1: (
+    rna: (
         <div role="document" id="rna-docs">
             <CIDCGithubMarkdown
-                path={`cidc-ngs-pipeline-api/master/rna/rna_level1.md`}
+                path={`cidc-ngs-pipeline-api/master/rna/rna.md`}
```

Comment:
```suggestion
                path={`cidc-ngs-pipeline-api/master/rna/RIMA.md`}
```

Reply:
I also changed it to rna.md in https://github.com/CIMAC-CIDC/cidc-ngs-pipeline-api/pull/11

Reply:
Ah, okay!